content/recipes/applied-examples/approach-cdisc.md

```diff
@@ -0,0 +1,167 @@
+* Auto-generated - some SDTM variables such as "STUDYID" or sequence ID ("-SEQ") will likely not be encoded directly in the dataset and need to be autogenerated or autocompleted during the data transformation process.
+
+
+### Putting together an ETL template
+
+For most of the scenarios listed above, with the exception of direct one-to-one matches, ETL scripts will need to be implemented in order to convert the data into an SDTM-compliant format. As the implementation of scripts was outside the scope of our mapping work, we instead implemented a template that could be used as a basis to generate the scripts. A copy of our template can be downloaded from [this URL](https://webdav-r3lab.uni.lu/public/datacatalog_documents//dataset/df780ac2-c79d-11ec-9b1d-acde48001122/APPROACH_CDISC-SDTM_mappings.xlsx).
+
+
+
+### Issues, pitfalls and caveats
+
+One major issue that we encountered with both our mapping projects was the absence of data that was not encoded in the data dictionary. The SDTM includes an entire area called the "Trial Design Model", which provides a structured way of representing all activities of a clinical trial, such as planned visits, treatment arms and inclusion and exclusion criteria. While this information is a key component of a completely SDTM-compliant dataset, it was not part of the data dictionary and was therefore not part of the mapping strategy. As a result, any ETL process covering the data alone would not have generated a fully SDTM-compliant dataset. 
+
+Another area of the SDTMIG we did not include in our mappings was the representation of relationships in the data, such as those between different records for a given subject (RELREC domain). These relationships need to be factored into any ETL procedures.
+
+
+---
+
+## Conclusion
+
+>  * We presented an overview of our approach to mapping project-specific data dictionaries to the CDISC-SDTMIG with a view to transforming the corresponding datasets to be SDTM-compliant.
+> 
+> * The CDISC-SDTM standard supports the interoperability between datasets due to its high level of standardisation, detailed modelling and widespread use.
+> 
+> * The high level of detailed knowledge of the standard required to successfully convert a dataset to SDTM presents a significant hurdle 
+
+### What should I read next?
+> * [Ontology mappings](../interoperability/selecting-ontologies.md)
+> * [Data dictionary](../interoperability/creating-data-dictionary.md)
+
+
```

Comment:
TODO/Suggestions:
List open source and commercial ETL tools:
-[bonobo](https://www.bonobo-project.org/) a python library for ETL
-[Pentaho Ketle](https://dev.socrata.com/connectors/pentaho-kettle.html)
-[Talend](https://www.talend.com/products/talend-open-studio/)
- any other obvious candidate?

Reply:
I've added a reference to the ETL process recipe in both the ETL section and under further reading.